spot_wrapper/wrapper.py

```diff
@@ -1195,6 +1200,24 @@ def _robot_command(self, command_proto, end_time_secs=None, timesync_endpoint=No
             self._logger.error(f"Unable to execute robot command: {e}")
             return False, str(e), None
 
+    def _manipulation_request(
+        self, request_proto, end_time_secs=None, timesync_endpoint=None
+    ):
+        """Generic blocking function for sending requests to the manipulation api of a robot.
```

Comment:
I don't think this function is blocking?

Reply:
You're right!